errors.dd

```diff
@@ -173,6 +173,10 @@ $(DL
 Using exceptions to handle errors leads to another issue - how to write
 exception safe programs. $(DPLLINK exception-safe.html, Here's how).
 
+D uses $(DPLLINK statement.html#TryStatement, try, catch, finally statements)
+to handle exceptions. You can raise Exceptions using throw, as in
+throw new Exception("oops!");
```

Comment:
I was hoping somebody who knows D would replace it with something much more
helpful.

On Mon, Apr 6, 2015 at 7:58 AM, Nick Treleaven <notifications@github.com>
wrote:

> In errors.dd
> <https://github.com/D-Programming-Language/dlang.org/pull/956#discussion_r27801917>
> :
>
> > @@ -173,6 +173,10 @@ $(DL
> >  Using exceptions to handle errors leads to another issue - how to write
> >  exception safe programs. $(DPLLINK exception-safe.html, Here's how).
> >
> > +D uses $(DPLLINK statement.html#TryStatement, try, catch, finally statements)
> > +to handle exceptions. You can raise Exceptions using throw, as in
> > +throw new Exception("oops!");
>
> Could wrap this statement in backticks for code formatting.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/D-Programming-Language/dlang.org/pull/956/files#r27801917>
> .
>